climada/entity/exposures/crop_production.py

```diff
@@ -82,17 +81,40 @@
             'firr': {'name': 'irrigated'},
             }
 
-"""Conversion factor weight to kcal.
-    Sources:
-        - Nuss and Tanumihardjo, 2010. https://doi.org/10.1111/j.1541-4337.2010.00117.x
-        - USDA Nutrient Database"""
-KCAL_PER_TON = {'mai': 365e4,
-                'ric': 360e4,
-                'whe': 339e4,
-                'soy': 446e4,
-                }
-
-# default:
+"""Conversion factor weight [tons] to nutritional value [kcal].
+Based on Mueller et al. (2021), https://doi.org/10.1088/1748-9326/abd8fc :
+
+"For the aggregation of different crops, we compute total calories, assuming
+net water contents of 12% for maize, spring and winter wheat, 13% for rice and
+9% for soybean, according to Wirsenius (2000) and caloric contents of the
+“as purchased” biomass (i.e. including the water content) of 3.56kcal/g for maize,
+2.8kcal/g for rice, 3.35kcal/g for soybean and of 3.34kcal/g for spring and
+winter wheat, following FAO (2001).” (Müller et al., 2021)
+
+Version 1: conversion factors for crop biomass "as purchased",
+    here applied as default for FAO-normalized production:
+    Production [kcal] = Production [t] * KCAL_PER_TON [kcal/t]
+"""
+
+KCAL_PER_TON = dict()
+KCAL_PER_TON['biomass'] = {'mai': 3.56e6,
+                           'ric': 2.80e6,
+                           'soy': 3.35e6,
+                           'whe': 3.34e6,
+                           }
+"""
+Version 2: conversion factors for crop dry matter as simulated by most crop models,
+    here applied as default for raw ISIMIP model yields and derived production values:
+    Yield [kcal] = Yield [t] * KCAL_PER_TON [kcal/t] / (1-net_water_content_fraction)
+"""
+KCAL_PER_TON['drymatter'] = {'mai': 3.56e6 / (1-.12),
+                             'ric': 2.80e6 / (1-.13),
+                             'soy': 3.35e6 / (1-.09),
+                             'whe': 3.34e6 / (1-.12),
+                             }
+
+
+# Default folder structure for ISIMIP data:
 #   deposit the landuse files in the directory: climada_python/data/ISIMIP_crop/Input/Exposure
 #   deposit the FAO files in the directory: climada_python/data/ISIMIP_crop/Input/Exposure/FAO
```

Comment:
It seems to me like the `FAO` directory is not really used any more. Furthermore, this should refer to the directory defined in the CONFIG:
```suggestion
#   deposit the landuse and FAO files in the directory: {CONFIG.exposures.crop_production.local_data}/Input/Exposure
```